tasks/arrayIncludes.js

```diff
@@ -0,0 +1,11 @@
+// - includes
+// create an array and check if this array includes 5
+var arrayNumber = [1, 2, 5, 8, 10];
+
+function ifItsFiveinArray(array) {
+    if (array.includes(5) === true) {
```

Comment:
`array.includes(5)` returns true
not needed check true === true

```
     // true                  === true
if (array.includes(5) === true) {
```

Reply:
just if (array.includes(5)) {`

Reply:
please write in one row
function ifItsFiveinArray(array) {
  return array.includes(5);
}